### ISSUE TITLE
hueDynamic

### ISSUE THREAD

**Issue by Pronown** [diyhue/diyHue#584]
**Describe the bug**

**Steps to Reproduce**

1. Open hueDynamic (Windows or Android)
2. Link Bridge
3. Choose Palette
4. Choose Color
5. Choose Saturation

**Expected behavior**

Color should change.

**Logs**

```
2021-02-11 19:28:33,855 - root - INFO - [
    {
        "success": {
            "/lights/15/state/bri": 254
        }
    },
    {
        "success": {
            "/lights/15/state/hue": 53100
        }
    },
    {
        "success": {
            "/lights/15/state/sat": 78
        }
    },
    {
        "success": {
            "/lights/15/state/on": true
        }
    },
    {
        "success": {
            "/lights/15/state/transitiontime": 3
        }
    }
]
2021-02-11 19:28:33,859 - root - INFO - [
    {
        "success": {
            "/lights/13/state/bri": 254
        }
    },
    {
        "success": {
            "/lights/13/state/hue": 53100
        }
    },
    {
        "success": {
            "/lights/13/state/sat": 78
        }
    },
    {
        "success": {
            "/lights/13/state/on": true
        }
    },
    {
        "success": {
            "/lights/13/state/transitiontime": 3
        }
    }
]
XX.XX.63.3 - - [11/Feb/2021 19:28:33] "PUT /api/760375b86cd111eba576b827eb4c8352/lights/14/state HTTP/1.1" 200 -
2021-02-11 19:28:33,922 - root - INFO - in PUT method
2021-02-11 19:28:33,923 - root - INFO - /api/760375b86cd111eba576b827eb4c8352/lights/14/state
2021-02-11 19:28:33,923 - root - INFO - b'{"bri":254,"hue":53100,"sat":78,"on":true,"transitiontime":3}'
2021-02-11 19:28:33,926 - root - DEBUG - Sending M-Search response to XX.XX.63.2
2021-02-11 19:28:34,229 - root - INFO - [
    {
        "success": {
            "/lights/14/state/bri": 254
        }
    },
    {
        "success": {
            "/lights/14/state/hue": 53100
        }
    },
    {
        "success": {
            "/lights/14/state/sat": 78
        }
    },
    {
        "success": {
            "/lights/14/state/on": true
        }
    },
    {
        "success": {
            "/lights/14/state/transitiontime": 3
        }
    }
]
```

**Docker Info (please complete the following information):**
 - Architecture (Ex: x86, ARM): armv7l
 - Docker image version (Ex: latest, arm-05f043b, 128): 19.03.13
 - OS (Ex: Windows 10, Ubuntu Buster): Windows 10 or Android
 - Hardware (Ex: PC, Raspberry Pi 1/2/3/Zero): Raspberry Pi 3b+

**Checklist**
- [x] I confirm that this is a bug report
- [x] I confirm that I have completed this template to the best of my ability

**Additional context**

The lack of ability to change color renders the disco function useless.
THANK YOU for the wonderful add on and the for supporting it!

**Comment by Mevel** [comment]
What lights do you try to change color and how are they connected?

**Comment by Pronown** [comment]
Thank you for your reply. I am using Home Assistant to connect and have tried with a few different lights, including my esp8266 based light strips, connected via wled, tried bulbs flashed to tasmota via mqtt, and have tried my Signify Wiz connected two different ways, one is via ip which is how you do the new integration for Wiz, the other way I was doing it was by using smadtthings as a middle man and connected via webhook.
The webhook way used to work okay with the other HueEmulator, but the connection was unreliable for the bridge and kept giving errors. (Not sure if that helps)
________________________________

**Comment by Pronown** [comment]
Well, I've been messing around with it and I also noticed that it will link to Hue Sync on PC, but won't connect. It just goes back to my actual bridge when I try.

**Comment by Mevel** [comment]
Any Success?

**Comment by Pronown** [comment]
Sorry for late reply.
I had actually given up with the Wiz Integrated bulbs, went ahead and built a Neopixel using your instructions and actually was able to get that running beautifully with HueSync, so I stopped trying Dynamic.
Since last night I think, it stopped functioning. The Hue app said the bulbs needed an update. So I went into Home Assistant and updated the DIYHUE bridge. Now the Hue App doesn't see the lightstrip at all. I can connect via the IP, and I have gone ahead and done the Ping while searching, 0 packets lost. Any ideas?
